deprecations/README.md

```diff
@@ -35,6 +35,7 @@ The table columns have the following meanings:
 
 | Feature | Comments | Deprecated | Discontinued |
 | :-- | :-- | :-- | :-- |
+| `Entities`<br>in [General webhook request body](../webhooks/wh-general.md#request-body) | Fetch data with corresponding GetAll endpoint | 22 Aug 2022 | - |
```

Comment:
Move to 23, link the new table

Reply:
I can't review this particular change, because the rich diff is all over the place, it's like trying to review Matrix code, ha.